README.md

```diff
@@ -1,27 +1,23 @@
-# Datalogics PDF Java Toolkit Sample repository
+# Datalogics PDF Java Toolkit Sample Repository
 
 This repository contains samples for use with [Datalogics PDF Java Toolkit](http://www.datalogics.com/products/pdf/pdfjavatoolkit/).
 
-Please note that even though the samples are MIT licensed, you still require
-a license from Datalogics for PDF Java Toolkit in order to run these samples. Please
-sign up for an evaluation [here](http://www.datalogics.com/products/pdf/pdfjavatoolkit/eval/)
-or [contact us](http://www.datalogics.com/company/contact-us/) to learn more before
-evaluating Datalogics PDF Java Toolkit.
+Please note that even though the samples are MIT licensed, you still require a license from Datalogics for PDF Java Toolkit in order to run them. Please sign up for an evaluation [here](http://www.datalogics.com/products/pdf/pdfjavatoolkit/eval/) or [contact us](http://www.datalogics.com/company/contact-us/) to learn more before evaluating the product.
 
 ## Requirements
 
 * Java SE 1.7
 * Maven 3.3.9
 
-## Using with an evaluation version of PDFJT
+## Using with an evaluation version of PDF Java Toolkit
 
-The evaluation version of PDFJT has license management, and a different artifact name: ``pdfjt-lm``. There's also a corresponding ``talkeetna-lm`` which similarly depends on ``pdfjt-lm``. Switching to use these versions of PDFJT and Talkeetna is provided with Maven profiles.
+The evaluation version of PDF Java Toolkit has license management, and a different artifact name: ``pdfjt-lm``. There's also a corresponding ``talkeetna-lm`` which similarly depends on ``pdfjt-lm``. Switching to use these versions of PDF Java Toolkit and Talkeetna is provided with Maven profiles.
```

Comment:
These are good! I like using the name of the product instead of the abbreviation!